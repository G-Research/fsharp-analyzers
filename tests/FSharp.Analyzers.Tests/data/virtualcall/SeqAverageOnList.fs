module M

let f () =
    let myList = [ 1.0 .. 10.0 ]
    Seq.average myList // should warn
