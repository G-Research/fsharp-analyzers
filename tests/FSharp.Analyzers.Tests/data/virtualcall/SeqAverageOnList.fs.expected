GRA-VIRTUALCALL-001 | Warning | (5,4 - 5,22) | Consider replacing the call of Seq.average with a function from the List module to avoid the costs of virtual calls. | [(5,4 - 5,7) List]
